incubator/chartmuseum/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
 name: chartmuseum
-version: 0.3.6
```

Comment:
Lets put 0.3.7 here.
Semvar in charts is strange to grok, but I think updating Docker image should be a patch bump

Reply:
If you're set on that, I'll go with it. I maintain a couple other charts and have had to deal with this a lot. If you don't mind considering an alternative perspective, here it is...

The Docker image you're bumping contains a new minor revision (i.e. new features), then implicitly, the chart _also_ contains new features-- features which must sometimes be accounted for in the chart, as evidenced by your request for me to include the following in the chart (which I am happy to do, btw):

```
STORAGE_ALIBABA_BUCKET
STORAGE_ALIBABA_PREFIX
STORAGE_ALIBABA_ENDPOINT
STORAGE_ALIBABA_SSE
```

Sleep on it. Ultimately, I'm fine with whatever you decide is right here.

Reply:
@krancour ok, I think youre right :)

Lets keep with 0.4.0. Will use this reasoning going forward

Reply:
sgtm. Thanks for listening.

Just added the Alibaba detail as well.